azure-mgmt-containerregistry/src/main/java/com/azure/management/containerregistry/implementation/BuildArgumentImpl.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+package com.azure.management.containerregistry.implementation;
+//
+//import com.microsoft.azure.management.apigeneration.LangDefinition;
+//import com.microsoft.azure.management.containerregistry.Build;
+//
+//import java.util.ArrayList;
+//
+/**
+ * Implementation for container group's volume definition stages interface.
+ */
+public class BuildArgumentImpl {
+
+}
+//@LangDefinition
```

Comment:
Is this already commented in track 1?

Reply:
You can see the commit, all track1 is from the first commit.

Reply:
Yes, after I check for the commit. The BuildArgumentImpl is commented from track 1.